pkg/testutils/sensors/sensors.go

```diff
@@ -44,21 +44,41 @@ type TestSensorManager struct {
 // unqique directory for maps/etc, and will also register the necessary cleanup functions using
 // t.Cleanup()
 func GetTestSensorManager(ctx context.Context, t *testing.T) *TestSensorManager {
-	if mgr := observer.GetSensorManager(); mgr != nil {
+	return getTestSensorManager(ctx, t, false)
+}
+
+func GetTestSensorManagerWithDummyPF(ctx context.Context, t *testing.T) *TestSensorManager {
+	return getTestSensorManager(ctx, t, true)
+}
+
+func getTestSensorManager(ctx context.Context, t *testing.T, useDummyPF bool) *TestSensorManager {
```

Comment:
Instead of a `bool`, I think it would be better to pass the interface implementation. WDYT?

Reply:
Thanks, I changed it to pass `policyfilter.State` instead